Autopilot/Inc/SensorFusion.hpp

```diff
@@ -0,0 +1,60 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong, Dhruv Rawat, Anthony Bertnyk
+ */
+
+#include "AttitudeDatatypes.hpp"
+#include "fetchSensorMeasurementsMode.hpp"
+
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+
+typedef struct {
+    float IMUroll, IMUpitch, IMUyaw; //rad
+    float IMUrollrate, IMUpitchrate, IMUyawrate; //rad/s
+    float Airspeed; //m/s
+} SFAttitudeOutput_t ;
+
+typedef struct {
+    float altitude; //m
+    float altSpeed; //m/s
+    long double latitude; //Decimal degrees
+    float latSpeed; //m/s
+    long double longitude; //Decimal degrees
+    float longSpeed; //m/s
+} SFPositionOutput_t;
+
+const int NUM_KALMAN_VALUES = 6; 
+
+typedef struct {
+    float prevX[NUM_KALMAN_VALUES];
+    float prevP[NUM_KALMAN_VALUES*NUM_KALMAN_VALUES];
+} SFIterationData_t;
```

Comment:
Please remove the typedefs. It really isn't needed for struct declarations in c++